cluster_management/src/main/java/com/pinterest/rocksplicator/ConfigGenerator.java

```diff
@@ -300,23 +420,34 @@ private void generateShardConfig() {
     this.dataParameters.remove("content");
     this.dataParameters.put("content", newContent);
     HttpPost httpPost = new HttpPost(this.postUrl);
+
+    long shardPostingTimeMillis = System.currentTimeMillis();
+
     try {
       httpPost.setEntity(new StringEntity(this.dataParameters.toString()));
       HttpResponse response = new DefaultHttpClient().execute(httpPost);
       if (response.getStatusLine().getStatusCode() == 200) {
         lastPostedContent = newContent;
-        LOG.error("Succeed to generate a new shard config, sleep for 2 seconds");
-        TimeUnit.SECONDS.sleep(2);
```

Comment:
we do not need the delay anymore?

Reply:
Why should there be any delay here? Once it is successfully posted... we should be introducing artificial delay... Since it is not documented here as to why this delay has been introduced.. better to remove it.. as it adds to unnecessary 2 seconds delay in processing of next notifications.